superset/viz.py

```diff
@@ -789,10 +789,13 @@ def get_data(self, df):
         data = {}
         records = df.to_dict('records')
         for metric in self.metric_labels:
-            data[metric] = {
-                str(obj[DTTM_ALIAS] / 10**9): obj.get(metric)
-                for obj in records
-            }
+            values = {}
+            for obj in records:
```

Comment:
Let's duplicate less logic, something like:
```python
v = obj[DTTM_ALIAS]
if hasattr(v, 'value'):
    v = v.value
values[str(v / 10**9)] =  obj.get(metric)
```

Reply:
yeah that's much cleaner 👍